tests/Examples/lattigo/bgv/dot_product_8_sk/BUILD

```diff
@@ -9,7 +9,7 @@ heir_lattigo_lib(
     name = "dot_product_8_sk",
     go_library_name = "dotproduct8sk",
     heir_opt_flags = [
-        "--mlir-to-bgv=ciphertext-degree=8 use-public-key=false",
+        "--mlir-to-bgv=ciphertext-degree=8 use-public-key=false encryption-technique-extended=true",
```

Comment:
Should this be true by default, even for OpenFHE (maybe with an error when it's not properly set for OpenFHE+{BGV, CKKS})? Or maybe have a wrapper pipeline that sets it to true by default for lattigo...

I just worry that lattigo users will not know to set this option if it's not the default.

Reply:
I think this should be addressed by #1420, but did not get a immediate plan on how to implement that